polyphemus/lib/ipi/migrations/01_flow_model_migration.rb

```diff
@@ -0,0 +1,161 @@
+require 'etna'
+
+class IpiAddFlowModelMigration
```

Comment:
Seems like this migration would blow up in any environment that did not have the ipi model already configured?  Thus this migration would break, for instance, bringing up a new development environment.

I'd feel more safe with this being an optional script that can be run manually, or if it can check some precondition before running (remember, this could even be run just as the magma environment is booting, and no api is even available yet).